Readme.md

```diff
@@ -0,0 +1,11 @@
+# ROS Vision Messages
```

Comment:
Move the readme from the vision_msgs package into the root after this reorganization and add a section about the rviz plugins + that lovely gif you made

Reply:
Remove this file's contents

Reply:
Implemented the structure changes